### PR TITLE
PLAT-736 Added IDomEnableable.isDisabled() and IDomTextualInput.isReadonly()

### PR DESCRIPTION
**Pull request by oliver-richers**
Test Plan:
- start TestCasesDiv
- click on the test buttons to disable/enable the input
- click on the test buttons to readonly/read-write the input

image

### CODE REVIEW

File: platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java [hunk 1]
```diff
@@ -88,4 +88,28 @@ default void select() {
...
 		getDomEngine().select(this);
 	}
+
+	// -------------------------------- read-only -------------------------------- //
+
+	/**
+	 * Makes this {@link IDomTextualInput} read-only.
+	 * <p>
+	 * In contrast to {@link #setEnabled(boolean)}, the user can still see and
+	 * copy the text of this {@link IDomTextualInput}.
```
Comment (thees-k) [inline platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java:98]: I tested with Firefox. You _can_ copy the text of a disabled input field, too. So this Javadoc comment is not right.

**Review by thees-k — CHANGES_REQUESTED** [review]
I tested `TestCasesDiv` with Firefox.

The only difference between "read only" and "disable" seems to be that the text of a disabled input field is much more transparent.

Maybe we don't need "read only" at all?

File: platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java (hunk 1 above)
Comment (oliver-richers) [reply platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java:98]: interesting

Comment (oliver-richers) [reply platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java]: I removed this comment.

**Review by thees-k — APPROVED** [review]

**Pull request merged**